### DISCUSSION TITLE
How to get infinite on mobile without synchTouch

### DISCUSSION THREAD

**Discussion by gabrielemotter** [darkroomengineering/lenis#318] [Q&A] [answered]
Hey there!
Is it possible to achieve infinite scroll on mobile without using the synchTouch? Or what are the best setting to achieve a pleasant highjacked scroll?
So far I'm not happy with the experience that the synchTouch is giving me, but I also need to enable infinite scroll that doesn't seem to pick up the touch move event on mobile without the synchTouch

Thank you so much in advance!

**Comment by clementroche** [comment, marked as answer]
That's not possible to enable infinite scrolling without syncTouch
